corehq/apps/app_manager/tests/test_bulk_ui_translation.py

```diff
@@ -80,15 +80,17 @@ def test_translation(self):
             }
 
         )
-        self.assertEqual(len(error_properties), 4)
+        self.assertEqual(len(error_properties), 2)
         self.assertEqual([e.strip() for e in error_properties], [
-            "Property updates.found should contain ${0}, ${1} but en value contains no parameters.",
-            "Property updates.found should contain ${0}, ${1} but fra value contains ${0}, ${1}, ${2}.",
             "Could not understand '${x0}' in fra value of sync.progress.",
             "Could not understand '$ {1}' in fra value of sync.progress.",
         ])
-        # There should be a warning that 'unknown_string' is not a CommCare string
-        self.assertEqual(len(warnings), 1, warnings)
+        self.assertEqual(len(warnings), 3)
+        self.assertEqual([e.strip() for e in warnings], [
+            "unknown_string is not a known CommCare UI string, but we added it anyway",
+            "Property updates.found should contain ${0}, ${1} but en value contains no parameters.",
+            "Property updates.found should contain ${0}, ${1} but fra value contains ${0}, ${1}, ${2}.",
```

Comment:
nit: Should that be "Property updates - ", i am guessing its a message for property updates?

Reply:
Not exactly, "updates.found" is the name of the property. But it would be good for the two messages to be more consistent. c6bf44a868a

Reply:
Aha! may be a single quote around that would help.

Reply:
sure, c8a381395bc

Reply:
Thanks